examples/pull/kubernetes/README.md

```diff
@@ -0,0 +1,104 @@
+# Pyroscope pull mode and Kubernetes Discovery
+
+This example demonstrates how Pyroscope can be used to scrape pprof profiles from Kubernetes pods.
+
+### 1. Run minikube kubernetes cluster locally (optional)
+
+In this example we use minikube for sake of simplicity, visit [minikube documentation](https://minikube.sigs.k8s.io/docs/start/)
+to learn more.
+
+```shell
+minikube start
+```
+
+### 2. Install Pyroscope with Helm chart
+
+The official [Pyroscope Helm chart](https://github.com/pyroscope-io/helm-chart) deploys Pyroscope server and creates proper RBAC roles:
+
+```shell
+helm repo add pyroscope-io https://pyroscope-io.github.io/helm-chart
+helm install demo pyroscope-io/pyroscope -f values.yaml
+```
+
+Note that we apply configuration defined in `values.yaml`: Pyroscope uses exactly the same discovery mechanisms as
+Prometheus does in order to ensure smooth user experience, and it fully supports
+[Kubernetes Service Discovery](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#kubernetes_sd_config):
+
+<details>
+    <summary>values.yaml</summary>
+
+```yaml
+---
+pyroscopeConfigs:
+  log-level: debug
+  scrape-configs:
+  - job-name: 'kubernetes-pods'
```

Comment:
Why not make this snake case? Like `job_name`, `kubernetes_sd_configs` etc

Reply:
It would be good to keep it consistent

Reply:
@yeya24 Honestly, this is just the convention we started with in our main config.

@kolesnikovae How hard do you think it would be to support both kebab-case and snake-case?